package/MDAnalysis/analysis/hole2/hole.py

```diff
@@ -228,7 +229,7 @@ def hole(pdbfile,
         warnings.warn(msg.format(output_level))
 
     # get executable
-    exe = util.which(executable)
+    exe = shutil.which(executable)
```

Comment:
not looking to add tests to cover these if the migration is also being done in hole2-mdakit